timeseries/src/autogluon/timeseries/models/__init__.py

```diff
@@ -13,6 +13,8 @@
 
 __all__ = [
     "DeepARModel",
+    "DLinearModel",
```

Comment:
I am curious if there is a particular reason or quantitative analysis supporting the inclusion of these two models